pyinfra/modules/iptables.py

```diff
@@ -0,0 +1,65 @@
+# pyinfra
+# File: pyinfra/modules/iptables.py
+# Desc: manage iptables rules
+
+'''
+The iptables modules handles iptables rules
+'''
+
+from __future__ import unicode_literals
+
+from pyinfra.api import operation
+
+
+@operation
+def forward(state, host, dport, to_ip, to_port=None,
```

Comment:
Sorry, @hoh I meant keeping the same filename but just renaming the function to `nat`, such that the operation used in deploys is `iptables.nat()`...